apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/SimpleManagedTracerTest.cs

```diff
@@ -0,0 +1,256 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Devtools.Cloudtrace.V1;
+using Xunit;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Google.Protobuf.Collections;
+
+namespace Google.Devtools.AspNet.Tests.CloudTrace
+{
+    public class SimpleManagedTracerTest
+    {
+        private const string TraceId = "trace-id";
+        private const string ProjectId = "project-id";
+
+        private static readonly ITraceConsumer UnusedConsumer = (new Mock<ITraceConsumer>()).Object;
+        private static readonly Dictionary<string, string> EmptyDictionary = new Dictionary<string, string>();
+        private static readonly StackTrace EmptyStackTrace = new StackTrace();
+
+        private static Cloudtrace.V1.Trace CreateTrace(string projectId = ProjectId, string traceId = TraceId)
```

Comment:
Nit - as a matter of style, I'd make this an expression-bodied member and remove the `()` after `new Cloudtrace.V1.Trace`. (Do we have our own `Trace` class here that's stopping us from just using `new Trace()`? I'm struggling to keep up...)